### PR TITLE
Refactor 4-line if-statement for clarity

### PR DESCRIPTION
**Pull request by clemens-tolboom**
## Description

This patch make the code a little more readable and removes two vars in the if statement. Not much.
## Motivation and Context

It took me a while to understand the former notation

``` javascript
if (circle1.x + bothRadii > circle2.x &&
    circle1.y + bothRadii > circle2.y &&
    circle1.x < circle2.x + bothRadii &&
    circle1.y < circle2.y + bothRadii) {

# refactors to

if (circle1.x - circle2.x + bothRadii > 0 &&
    circle1.y - circle2.y + bothRadii > 0 &&
    circle1.x - circle2.x - bothRadii < 0 &&
    circle1.y - circle2.y - bothRadii < 0) {

# which naturally introduces dx and dy

```
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bot.user.js [hunk 1]
```diff
@@ -273,13 +273,12 @@ var canvas = window.canvas = (function () {
...
+            var dx = circle1.x - circle2.x;
+            var dy = circle1.y - circle2.y;
 
             // Pretends the circles are squares for a quick collision check.
             // If it collides, do the more expensive circle check.
-            if (circle1.x + bothRadii > circle2.x &&
-                circle1.y + bothRadii > circle2.y &&
-                circle1.x < circle2.x + bothRadii &&
-                circle1.y < circle2.y + bothRadii) {
+            if (dx + bothRadii > 0 && dy + bothRadii > 0 && dx - bothRadii < 0 && dy - bothRadii < 0) {
```
Comment (ChadSki) [inline bot.user.js]: Should probably use a linebreak again.

```
if (dx + bothRadii > 0 && dy + bothRadii > 0 &&
    dx - bothRadii < 0 && dy - bothRadii < 0) {
```

Much easier to understand, thanks for shortening from 4 lines.

Comment (ermiyaeskandary) [inline bot.user.js]: @clemens-tolboom Please resolve hound comment by doing ^^^^^ and then we can merge! Good job!

**Comment by clemens-tolboom** [comment]
Added the line break. Not sure a forced push was the best and whether 2 commits is ok.

**Comment by tjorim** [comment]
Well done, thanks! :+1: LGTM

**Comment by ermiyaeskandary** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/249/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ChadSki** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/249/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Pull request merged**